stories/jsx-import-source-pragma.tsx

```diff
@@ -0,0 +1,35 @@
+/** @jsxImportSource @compiled/react */
+
+export default {
```

Comment:
Should these be stories or type tests? There are type tests for flow, you could use a similar format for typescript.

Reply:
Both - there are tests testing these flows. They are also stories so it makes sure the runtime is actually set (because it imports from `@compiled/react/jsx-runtime`).